src/java.desktop/macosx/native/libawt_lwawt/awt/a11y/CommonComponentAccessibility.m

```diff
@@ -349,6 +349,9 @@ - (void)postSelectedTextChanged
 - (void)postSelectionChanged
 {
     AWT_ASSERT_APPKIT_THREAD;
+    if ([self respondsToSelector:NSSelectorFromString(@"invalidateSelectionCache")]) {
```

Comment:
You can override this method for the tree. to avoid checking the selector support... This is a time-consuming procedure.

Reply:
Sure.